Kitodo/src/main/java/org/kitodo/services/command/CommandService.java

```diff
@@ -52,7 +52,7 @@ public CommandResult runCommand(String script) throws IOException {
 
         CommandResult commandResult = commandInterface.runCommand(generateId(), script);
         if (commandResult.getMessages().get(0).contains("IOException")) {
-            throw new IOException();
+            throw new IOException(commandResult.getMessages().get(1));
```

Comment:
You are sure about using the second message are containing a useful / meaningful message if the first one is used for checking?